examples/longhorn/README.md

```diff
@@ -25,6 +25,37 @@ components:
   - name: longhorn
     required: true
     description: "Deploy Longhorn into a Kubernetes cluster.  https://longhorn.io"
+    actions:
+      # Run the Longhorn Environment Check on this cluster's nodes.
+      onDeploy:
+        before:
+          - env: 
+            - "PATH=$PATH:./"
+          - cmd: ./environment_check.sh
+      # Set the delete confirmation flag for Longhorn
+      onRemove:
+        before:
+          - env:
+            - "PATH=$PATH:./"
+          - cmd: "kubectl -n longhorn-system patch -p '{\"value\": \"true\"}' --type=merge lhs deleting-confirmation-flag"
```

Comment:
do you still need the path stuff here if we have `./zarf tools kubectl` vendored in now?

Reply:
Removed kubectl and used the vendored version.  I missed that kubectl had been vendored in.  I had to alias it because otherwise it didn't handle the -n properly

```
[kuber@k3s80controller1 longhorn]$ zarf package remove longhorn-example --confirm

Saving log file to /tmp/zarf-2023-02-24-21-21-27-2390121056.log
  ✔  Completed command ""
     Error: unknown shorthand flag: 'n' in -n
     Usage:
       zarf tools [command]

     Aliases:
       tools, t

     Available Commands:
       archiver         Compress/Decompress generic archives, including Zarf packages.
       clear-cache      Clears the configured git and image cache directory.
       gen-pki          Generates a Certificate Authority and PKI chain of trust for the given host
       get-creds        Display a Table of credentials for deployed components. Pass a component name to get a single credential.
       monitor          Launch a terminal UI to monitor the connected cluster using K9s.
       registry         Tools for working with container registries using go-containertools.
       sbom             Generates a Software Bill of Materials (SBOM) for the given package

     Flags:
       -h, --help   help for tools

     Global Flags:
       -a, --architecture string   Architecture for OCI images
           --insecure              Allow access to insecure registries and disable other recommended security enforcements. This flag should only be used if you have a specific reason and accept the reduced security posture.
       -l, --log-level string      Log level when running Zarf. Valid options are: warn, info, debug, trace (default "info")
           --no-log-file           Disable log file creation
           --no-progress           Disable fancy UI progress bars, spinners, logos, etc
           --tmpdir string         Specify the temporary directory to use for intermediate files
           --zarf-cache string     Specify the location of the Zarf cache directory (default "~/.zarf-cache")

     Use "zarf tools [command] --help" for more information about a command.

     Error: unknown shorthand flag: 'n' in -n
     ERROR:  Unable to remove the package with an error of: &fmt.wrapError{msg:"unable to remove the component
             (longhorn): unable to run the before action for component (longhorn): command \"zarf tools kubectl
             -n longhorn-system patch -p '{\"value\":...\" failed after 0 retries",
             err:(*fmt.wrapError)(0xc000279d20)}
             
```